go.mod

```diff
@@ -165,6 +165,7 @@ require (
 	k8s.io/metrics v0.0.0
 	k8s.io/repo-infra v0.0.0-20181204233714-00fe14e3d1a3
 	k8s.io/sample-apiserver v0.0.0
+	k8s.io/system-validators v0.0.0-20191104102940-4fc9c1c0ccc1
```

Comment:
/hold
we need to have a tag in the new repo first.

there is also this PR that we might want to merge in there first:
https://github.com/kubernetes/system-validators/pull/3/files

Reply:
@yastij 
https://github.com/kubernetes/system-validators/releases/tag/v1.0.0

Reply:
+1 for the tag.
BTW, what versioning scheme do we employ for the system validators? Starting from `v1.0.0` is kind of strange for me.

Reply:
I was thinking more on the ground of k8s mirror versioning rather than starting from `1.0.0`. But anyway, it's OK like this too.